addons/gdUnit3/src/cmd/CmdCommandHandler.gd

```diff
@@ -0,0 +1,80 @@
+class_name CmdCommandHandler
+extends Reference
+
+var _cmd_options :CmdOptions
+var _command_func_refs :Dictionary
```

Comment:
add a comment for the data structure
# holds the command callbacks by key:<cmd_name>:String and value: [<cb single arg>, <cb multible args>]:Array